packages/jest-core/src/TestScheduler.ts

```diff
@@ -180,7 +181,7 @@ export default class TestScheduler {
           aggregatedResults.snapshot.filesRemoved)
       );
     };
-
+    reporterService.send('START');
```

Comment:
@thegoleffect how can we `await` for the `onRunStart` functions to finish running as we do in the line below? (L#185)
cc: @victorphoenix3 @jcs98

Reply:
if you want to "wait", you should have the async method dispatch another event on completion that transitions to another state. Then, replace the await code with either 

1) code that listens on states & executes something locally
2) code that registers a method to the context that will get executed if exists on completion/state transition

Reply:
Thanks, we were able to get it running!